### PR TITLE
Adds lesser magboot slowdown to Vox boots, adds stomp attack

### PR DESCRIPTION
**Pull request by ValkyrieSkies**
Adds some slowdown to the Vox magboots and allows them to use the stomp attack . A lot of people seem to agree that the Vox having no penalty magboots is just silly and unnecessary.

Compiled and tested successfully.

:cl:
* rscadd: Gives Vox magboots a slowdown less than standard magboots, and enables a customized version of the magboot stomp command.

### CODE REVIEW

**Comment by Durgen** [comment]
VIDF assemble

**Comment by ghost** [comment]
reminder that vox magboots have NO WEAKNESS

**Comment by jknpj** [comment]
Vox, hyperzine. Hyperzine, vox.

DON'T DARE YOU TO START STEALING THE OUTPOST'S CHEM DISPENSER AGAIN @Durgen

**Comment by IAGFPRMF** [comment]
SKREEEEEEEE

**Comment by jknpj** [comment]
I almost forgot, raiders need their snowflake boots now.

**Comment by Exxion** [comment]
If you really *have* to do this, at least give them back their stomp damage so they aren't strictly worse than normal magboots

Personally I think they should have slowdown, but less than standard magboots to make up for their rarity, restrictions, and inability to stomp

**Comment by ValkyrieSkies** [comment]
What's the standard non-magboot stomp damage? I'll give em a slightly higher one once I get home because metal talons.

**Comment by jknpj** [comment]
@Exxion they aren't magboots, why would they have magnetic boot stomp? They are literal clawing boots. 

Just being ZAS-proof is good enough for a race that can travel through space without using space worth gear.

**Comment by Riekland** [comment]
95fd27db02006ae13cedc182572342bbe7ca3fae

VERY nice pr

**Comment by Exxion** [comment]
45, but the message won't make sense

Also, I lied
I think optimally, they should not have slowdown, but should be worse at preventing stuns than normal magboots (for example, not preventing slips and making it so that you still get stunned by airflow but just not knocked down and pushed around (yes that is possible) (I think))
But someone would have to actually code that

@jknpj Vox being spaceproof on their own is pretty much an oversight
Just nobody's bothered fixing it
And yes, I agree that the magstomp doesn't make sense for them, but if you're just going to make the vox boots strictly worse than regular magboots, why would anyone use them? They're harder to get (unless you're a trader) and only work for vox.

**Comment by ValkyrieSkies** [comment]
I'm just talking about the damage for kicking someone on the ground, not the magboot stomp.

Also the very way Voxboots are described as working means it would be ridiculous not to have slowdown unless Vox had fucking enormous legs made of pure muscle.

**Comment by Durgen** [comment]
Clawboots are the only(?) race specific item that vox can use which humans can't. It's one of the interesting perks about traders which differentiate them from the other roles in the game. 

>  Also a huge amount of people seem to agree that the Vox having no penalty magboots is just silly and unnecessary.

Submitting a balance change because the discord had a trader hatefest today is bad form imo. You should have created an issue first to discuss this.

**Comment by ValkyrieSkies** [comment]
An issue would've just led to more useless squabbling that probably would've had no proper outcome regardless of what way it went. We have a discussion happening now and nothing has to be done in haste, but if an outcome is decided the follow through is ready to go and be merged or scrapped.

**Comment by Exxion** [comment]
@Durgen Nah, opening a PR is fine for discussion
Especially if it's for a small (in terms of effort to perform) change like this
Speaking of that

**Comment by Exxion** [comment]
Going to hold off on [Controversial] for now because I don't want to pull a Riri

**Comment by ValkyrieSkies** [comment]
As for a unique aspect, no slowdown is just kind of a shitty easy upside that doesn't fit the item thematically. Let's see something more unique, like a special claw based kick with slightly more DPS than mag stomp but less lethal per kick.

**Comment by ghost** [comment]
RIP AND TEAR (with clawboots)

**Comment by yclatious** [comment]
Its essentially a low to no tech shitty claw boot thing that entrenches itself into the ground as a means of securing yourself and not letting the ZAS train move you to wall banging town, vox are low tech, low respect race who rely on trading to get by, they don't get tons of high tech stuff and even the advanced Nanotrasen cheif engieneer boots, boots with supposed high tech magnetic research and development, with atleast twenty times more cash put into their rnd than vox boots have, still get more slowdown.

Vox traders can cry "VOX HATE METACLUB, WOOOOW HOW DARE PEOPLE FINALLY NERF THE THING EVERYONE KNOWS HAD NO REASON TO HAVE NO DOWNSIDES" but they have no ground to stand on, vox get way more shit than they should, including free space suits, a assortment of trader specific trade items they can get more than one out of by wire coining, free jetpacks, a shuttle and a completely seperate, fully functioning kind of botany and cuisine that's vox exclusive, not to mention that unless memory serves wrong, deff even has a outpost with a chem dispenser on it.

I get fun, I get vox are not made exclusively for shitting on but be sensible, this nerf should have happened ages ago. Vox have everything else going for them alongside having the most cushty and comfy job bar assistants who make booze dens, you can forego the no slowdown ones and get a slight slowdown ya big feathered babies.

**Comment by Dusk-a** [comment]
Why should their magclaws get their no slowdown removed? There are only a very limited amount possible and usually its only the people who shouldn't be doing much involving the station that have them. Sure they can sell 3 or 4 pairs but who are they going to? Most likely not security because who plays security vox? It will be people like doctors or engineers who will have no slow magboots that can help save the station when they need it or people in civilian jobs where it won't really matter if they have it or not.

**Comment by Aquaes** [comment]
Why would you give them a better kick, really? I mean yeah, it'd make sense, but god damn it'd be the most useless shit for the trader of all roles. They're not something that's meant to be getting in combat much at all, and I don't know why on earth you'd encourage that. No slowdown at least makes them useful for dicking around in space, which is something traders wind up doing not infrequently. Yes, it might not make logical sense but it's a good fit for the job.

**Comment by yclatious** [comment]
Limited amount argument is only valid when they are restricted in availability, vox traders get them as easy as a assistant gets all his tools, that's a really weak excuse to justify keeping in a nonsensical slowdown that should have been removed way ago.

Your excuse of selling them to station and station using it as heroes is also bullshit as its highly circumstantial and usually docs don't even leave medbay that much, your argument of trader birds who rush vaults is also weak as hell seeing as how then its a clearly outright better powergaming weapon thats inherently better than human tools, both in ease of access, efectiveness and availability, making the "nerf it because its blatantly better than it should be for no reason" argument even stronger.

No devil, what you just said is a bunch of weak arguments that are just diversions from the core issue or strengthen our argument. Its busted, better than humans, doesn't fit established rules and makes no sense, we dont even need 2 reasons to justify the PR, this is 4.

Enough, take the nerf like you should have ages ago and be sensible instead of trying to weasel out of it.

**Comment by ValkyrieSkies** [comment]
So should we make all magboots no slowdown for medical and engineering crew then? Come on man, the point you're trying to make is devoid of any real substance or actual explanation why something should be one way or the other. The point of this PR is to address a matter of imbalance and a fluff/crunch mismatch, and to suggest a better gimmick to give the boots than just being superior to standard magboots in every way.

**Comment by Durgen** [comment]
@ValkyrieSkies 

>no slowdown is just kind of a shitty easy upside that doesn't fit the item thematically. 

I would say that Trader is almost the canonical space exploration job, and having upgraded magboots helps in this regard. Perhaps Clawboots being fast is slightly immulsion breaking, but you could say that about lots of items. 

Do you think they are being abused somehow? IIRC when trader exosuits got nerfed it was mostly because station vox were going to the outpost and stealing/buying them off of traders and then being a nuisance. This just seems like a nerf, for the sake of nerfing.

Also Vox being able to wear magboots makes chimeras interesting, and allows vox to do leg swaps on humans. This change would dishonor the legend of Tinky the Uberclown.
f4c533942a20864b7e4385c1e7b9703cba05b8edc8464b5f71bd2c8a88ee8228

**Comment by ValkyrieSkies** [comment]
Fucking touchscreens.

**Comment by Exxion** [comment]
>Superior to standard magboots in every way

They're superior in one way (albeit a big one) and inferior in every other

**Comment by yclatious** [comment]
>vox get busted item
>vox use said busted item to spacewalk and be vault hunters
>because vox did this a lot, this is somehow a strong reason to justify busted item staying and for it all to continue fine and dandy despite all reasons listed  as to why its bad and a general "nerf when" reputation about said item having existed for a substancial ammount of time

That doesn't seem like a really good reason my man, they have a place as maybe bird ops but for normal traders, no, its a really dumb item that's busted and breaks crunch and fluff, I get its fun, but a new no slowdown magboots can be made with supposed ease, so stuff like Tinky can exist just fine anyhow.

I get why people defend it but it has valid and strong reasons to be nerfed lads.

**Comment by ValkyrieSkies** [comment]
Lets have some suggestions more thematic to their uses then. 

What about a stronger kick-off from surfaces to space like how using the superfart away from a wall in space sends you drifting at an accelerated rate? Let's hear some ideas, this is theperfect place for it.

**Comment by yclatious** [comment]
Leaving behind destroyed floor tiles or damaging exposed plating when moving would be perfectly acceptable alongside fitting in nicely with their tech level just being "claw the fuck out the floor so the wind wont take you", your sugestion is fine and adding in a different magboot stomp wouldnt hurt anyone.

**Comment by Exxion** [comment]
My suggestion (which I wrote above, but more poorly than this):
No slowdown and prevents slipping in space. When used in a vacuum, pretty much how they are now.
Also prevents airflow pushing you, but does not prevent the stun. Either still knocks you down or doesn't knock you down but immobilizes you for a moment; not sure which.

This would still make them excellent for space exploration, but much less useful on the station.

**Comment by yclatious** [comment]
Lad, you already have, to my memory, a perfectly fine looking jetpack roundstart with no complications, about 3 if you are alone, free space suit, gear like flashes, bolas, the works alongside innate pressure resistance so space doesn't even fuck you that badly, you can hang about with a trenchcoat while being in the void and shits fine and dandy and you still want even more by not having your perfect no slowdown no downside magboots when you even have access to a ton of canisters you can just drag alongside you for the sake of space jetpack refills on the fly, making you able to just fly about like its nobody's business?

And I though vox traders atleast tried to be humble since its part of their job, guess not.

**Comment by Kurfursten** [comment]
I don't get what makes the magboots "busted" anyway. These reactionary drama votes are dumb. Saying "we should take away something unique because traders can get jetpacks at roundstart" doesn't make sense to me. It's like saying, "Mechanics can get a jetpack at roundstart (on Box, from EVA), they shouldn't have telecoms access anymore" - what? How are those connected?

Hating Vox in-game is an interesting roleplaying dynamic, taking it seriously out-of-character is autistic. Learn the difference between game and reality.

**Comment by Durgen** [comment]
I don't think anyone has actually articulated why this change should be made besides 'immulsions'. If the concern is truly immulsions, we could just as easily change these lines to something else.
```
"You relax your deathgrip on the flooring.")
"You dig your claws deeply into the flooring, bracing yourself."
```

Do you feel these boots are being abused or something?

**Comment by Exxion** [comment]
@yclatious Who are you even talking to in that last comment?

**Comment by jknpj** [comment]
>"You dig your claws deeply into the flooring, bracing yourself."
>Do you feel these boots are being abused or something?

Besides turning them on forever because why would you ever turn them off? There's no point. Might as well just remove the toogle.

**Comment by ValkyrieSkies** [comment]
@Kurfursten 
Please don't try to dismiss things with "DAE JUST HATE VOX???", and that's not what's happening here. I'm simply making a PR about a topic that's been complained about for months without any action ever being taken and one that I agree is not fitting thematically. I quite like the Vox as a species and I'm generally one of the first to jump to their defence.

@Exxion 
The lesser ZAS resistance is an interesting take and I think it'd be a good middle-ground but I still can't advocate for no slowdown. Consider this;
Lesser ZAS resist, but slightly less slowdown than standard magboots
Activation text: "You tense up and prepare to brace yourself at a moment's notice."
Airflow stun text: "You dig your claws into the ground as a gust of wind hits you."
De-activation text: "You relax your posture and release your talons."

**Comment by yclatious** [comment]
@Exxion You lad.

@Kurfursten  It isn't reactionary if it has solid arguments behind it, this isn't something unique or interesting, its just no slowdown magboots for no good reason, be it ic ooc or lorewise, this isn't vox hate either, I don't hate vox at all, hell, I defend them being a thing and a lot of good memories I have of my older playtimes came from vox and interacting with them, waving away all criticism for this based on it just being "vox haters" is just uncalled for. 

People have been calling for the nerf for ages, this isnt anything new, not by far, much less reactionary, the almost impossible happened, for once, the complaints became action and because of that, somehow, people who argue the pr has merit just get called vox haters, waved aside and called autistic for "hating vox ooc" instead of taking the arguments at face merit.

Again, the arguments ive seen to keep them in were, and correct me if wrong, "the item has been here for a while, a good sizeable ammount of people called for its fixing/rebalancing, the calls were ignored, people used said unfixed/unbalanced item and a playstyle grew of it, hence it should remain", which is just saying it is indeed broken but people have fun and as such should stay, and chimeras, easily fixable by making a new item, neither of these are good strong ones and neither deal with the clash of lore, lack of balance, ilogical existance and they lack to prove how it isnt balanced, mostly because they cant, because it is indeed unbalanced.

I expected better of you kurfursten, gotta be honest, just waving it all aside as vox hate is not something id expect of you. Fucking hell, for once I try to be argumentative, explain my points right, fully, clearly, try to lay out the issues, what fueled the pr and im just labeled and ignored as some "lmao ooc vox racism dude" with the whole "nuh uh dont nerf vox" force upvoting it,

**Comment by Exxion** [comment]
@ValkyrieSkies I'd say significantly less slowdown than normal magboots for such a significant downside, but that can be decided later if that is what we go with, since implementing it would be hellish until after my next ZAS update PR due to the fucking thousand different procs for determining what happens to  a mob in airflow

@yclatious Then you've been misinformed, as I have played Trader for a total of like two rounds, one of which was a mistake, and both before many of the other things you mentioned had been added

**Comment by yclatious** [comment]
Sorry bout the expectation then lad, sincerely, I tend to assume people who defend items like these have a history of using them extensively alongside the rest of em, sue me, higher chance of being right than wrong.

**Comment by Intigracy** [comment]
FYI the CE boots have no slowdown, idk why someone claimed it has more than these.

https://github.com/vgstation-coders/vgstation13/blob/f93c1e8763b8d64463d6140ca32508019c4f13d1/__DEFINES/game.dm#L18

**Comment by ghost** [comment]
The c*der club continues their balance philosophy of "different is bad let's just make everything a wonky human".

Because that worked so well for all these recent silicon changes.

**Comment by Exxion** [comment]
What are you talking about? This is only the author's third PR. Not exactly part of the """""coder club"""""

**Comment by Exxion** [comment]
Also I think I can justifiably do this now

**Comment by SonixApache** [comment]
birds deserve nothing good
free no-slowdown magboots with no downsides are fucking retarded

**Comment by CptWad** [comment]
Just have them do regular stomp damage but with sharpness

**Comment by FudgePucker** [comment]
:+1: x :100:
Fucking finally.
>vox trader smashes into research with me (the warden) standing right there. Deport him, and by deport I mean strip of hardsuit and launch out of the disposal chute in incenirator.

Fucking birds have had this PR coming for a long time.

>filthy vox starting shit with security. I drag him to the gas chamber, strip off headset, and buckle cuff him to the chair just to let him cool off. Oh and took his breath mask so he could get some fresh station distro air.

HUMAN PRIDE STATION WIDE

**Comment by Dusk-a** [comment]
Where in that did he use his magclaws for an advantage? Just sounds like a shitter trader and a shitter warden doing an unauthorized execution 🤔

**Comment by ValkyrieSkies** [comment]
@devlkiller0 They were clearly being facetious.

**Comment by ValkyrieSkies** [comment]
Alright, so I've given them a lesser slowdown than normal magboots and a reduced damage but slightly faster stomp attack. Thoughts?

File: code/modules/clothing/spacesuits/alien.dm [hunk 1]
```diff
@@ -140,10 +145,12 @@
 	if(src.magpulse)
 		src.clothing_flags &= ~NOSLIP
 		src.magpulse = 0
+		src.slowdown = NO_SLOWDOWN
 		to_chat(usr, "You relax your deathgrip on the flooring.")
 	else
 		src.clothing_flags |= NOSLIP
 		src.magpulse = 1
+		src.slowdown = mag_slow
```
Comment (SonixApache) [inline code/modules/clothing/spacesuits/alien.dm]: Why not just `MAGBOOTS_SLOWDOWN_MED` and remove the var?

**Comment by SonixApache** [comment]
How about same slowdown as normal magboots, add stomp but with less damage and sharpness?
Also, update OP.

File: code/modules/clothing/spacesuits/alien.dm (hunk 1 above)
Comment (DamianX) [reply code/modules/clothing/spacesuits/alien.dm]: `mag_slow` is defined in the parent, I guess it makes sense to reuse that

Comment (SonixApache) [reply code/modules/clothing/spacesuits/alien.dm]: Pretty sure both those vars are pointless, though I'm no coding hero.

**Comment by ValkyrieSkies** [comment]
@SonixApache 
1) Exxion seemed very against the idea of them having slowdown so I was trying to reach a middle-ground because I don't think them having no slowdown makes sense.
2) Is there a way to add sharpness with the way the on_kick() proc works?

**Comment by Durgen** [comment]
@ValkyrieSkies 
If you are going to try to go the middle ground route, I don't think greater kick damage is a good idea. Traders having deadly kicking boots would probably promote misbehavior.

**Comment by ValkyrieSkies** [comment]
They have the same DPS as standard magboots, just that they deliver a leser amount of damage slightly faster.

**Comment by Exxion** [comment]
If traders want deadly kicking, they can just take off the boots anyway
Vox have a significant bonus to kick damage when not wearing shoes
I don't remember exactly what it is though

Anyway, I think this is an acceptable middleground
Though I might recommend closing this PR and opening a separate one in order to gauge the reactions to it in its new form without the old ones making it confusing

**Comment by strawberrytan** [comment]
Nerfing the fun that comes with interchangeable body parts by homogenisation and pasteurization of neat little features

I prefer voxboots being different for the requirement of having unique feet

**Comment by SonixApache** [comment]
every passing day i wonder how hard it would be to just purge the shitbirds from the code

**Comment by strawberrytan** [comment]
Muh fun quirks of surgery

**Comment by PJB3005** [comment]
@yclatious FYI I think your "muh lore" arguments are weak as hell if you're not prepared to actually write lore and make them interesting. I doubt a lot of people would be against making vox *interesting*, but "worse than humans in every way" is not interesting.

Do I agree vox need changes? Sure.

**Comment by yclatious** [comment]
Only thing stopping me from writing lore is lazyness and nobody telling me to do so/telling me its okay to do so tbh.

Personally, they are interesting because they, for the most part, while underequiped and lack the general rights of the crew, act in ways that are interesting and can lead to some fun moments, despite them mostly being in rather inhospitable and aggressive places, for them atleast. The stations air is plasma to them, nobody really likes them or will defend them, sec is begging for them to do something slightly illegal so they can arrest and confiscate stuff, I find it interesting, the way it all plays out and how despite all of this, they tend to be good traders and make for some good interaction, so sue me if I bring out lore and other ingame image justifications when they suddenly have CE tier magboots for free with extra pairs to give out to all their station buddies when all trader spots aren't filled in.

**Comment by PJB3005** [comment]
Vox are OP and I totally agree. I'm just saying that the lore argument is weak at best.

**Comment by jknpj** [comment]
Plasmaman got a sick brute multiplier to balance their burn res AND got their shit slapped by celt's shitcode, making them unable to remove their suits for 3 seconds without dying a slow death.

Diona got a huge healing nerf to balance their previous hyperzine abuse making them either slow as balls tanks or wonky green humans with 4x burn multiplier.

Vox can't take a single nerf to their NO SLOWDOWN magboots without the whole metaclub crying their beaks out and screaming "MUH DIVERSITY" "MUH FUN" and "HATE NERF". It really makes you think.

**Comment by strawberrytan** [comment]
Nice strawman, it is not illegal to disagree with a proposed change and it does not mean there is a birdfurry agenda, either

I'm just saying if we make vox talonboots magboots but worse, then we're making a change to what utility fancy limb surgery has. I don't generally play trader I just have an opinion same way you have one mang. 

I want talonboots to be a sidegrade instead of a downgrade. I don't mind if they required some actual work to attain, like the trader coat or PDA instead of being free and numerous.

**Comment by jknpj** [comment]
I said it really early on: leave raiders and raiders only with no_slowdown clawboots. Traders don't deserve antagonist/head tier equipment.

**Comment by strawberrytan** [comment]
Thanks 4 making medbay more shit jknpj

Rip frankensteining for anything but aesthetics anymore

**Comment by jknpj** [comment]
Vox legs still have boosted kick power, still valid.

**Comment by yclatious** [comment]
Not being able to powergame a frankenstein abomination into having no slowdown with vox legs isnt going to kill frankensteining strawbery, and you know it wont, besides, you do frankensteining for the fun of it, not just for powergay.

**Comment by strawberrytan** [comment]
More difficult to obtain for something you can get from other races a lot easier, Vox parts are kinda shitty by themselves because unlike other parts you can use to add damage in combat you can't wear a hardsuit if you have one single bit of Vox in you. That was a tradeoff for talonboots that won't be worth anything if regular mags are just better because they're also way easier to obtain and more numerous than talons.

**Comment by strawberrytan** [comment]
I'm asking for the talons to not be nerfed into the dirt predicated on the idea that everything humans have needs to be the better version. Magboots fit everyone. Talonboots require Vox feet to equip.

I'm not against no slowdown being taken away and replaced with something else but so far the idea of +damage instead is really lazy and uninteresting as a change from pretty much any angle.

**Comment by jknpj** [comment]
Just having lesser_slowdown still make them as good as nuke op boots, what the fuck are you talking about?

**Comment by yclatious** [comment]
The nerf is a lesser slowdown anyways, it isnt that massive of a hit, and again, if you frankensteined only for the sake of powergaming no slowdown boots, you were doing it for a shit reason, unless I misread, this doesnt make the normal magboots way better relative to the post nerf vox ones, its a minor slowdown they are recieving, even if they should be getting a hard one, so your complaining about how its all "ruined" over a mild slowdown that should have always been there to begin with.

You can call extra damage boring all you want but it sure beats only having the nerf that shoyld have happened ages ago and is only now being made. They aren't getting nerfed to the dirt, they are getting nerfed like they should have been, where did you get the idea they were getting nerfed to the ground?

I swear, the second you even dare thinking about nerfing this, its like people sprouted out of the woodwork to defend it, with the only excuse being powergaming and with the idea that somehow they were getting nerfed to unthinkable levels.

**Comment by strawberrytan** [comment]
It comes across as talonboots being nerfed to full magboots slowdown from the OP. Maybe if things were worded a bit clearer there this wouldn't happen. Paranoid much?

**Comment by jknpj** [comment]
But the OP says lesser slowdown 🤔

**Comment by strawberrytan** [comment]
Some is vague as fuck as a descriptor.

It literally says "Gives Vox magboots the same slowdown as standard magboots."

And I interpreted the 'lesser magboots slowdown' in the title to mean "lesser magboots" "slowdown" as in, slow generic magboots because better ones exist.

**Comment by yclatious** [comment]
Probably outdated description then.

**Comment by ValkyrieSkies** [comment]
This gives them a slowdown value of 1.75 instead of the standard 2.33. They're still faster than standard magboots.

**Comment by ValkyrieSkies** [comment]
You can see the exact changes on the changes tab.

**Comment by strawberrytan** [comment]
Shitty meme bodies continue to be possible to make without being completely worthless from a functional perspective eventhoughhyperzineexistsbutdoesalmostthesamething, hooray

**Comment by Durgen** [comment]
close w h e n

**Comment by FudgePucker** [comment]
Why the fuck do traders need zero down speed mag boots anyways?

**Comment by Sooose** [comment]
The change isn't that bad for regular vox but don't do this to raiders. The last thing they need with their pressure suits slowing them down more than any other hardsuit is slower boots. Don't claim remove the pressure suit either, they can't do that without clothing to keep them warm in space and a pocket tank.

**Comment by Aquaes** [comment]
Why is anybody treating this like a racial balance thing? It's not a matter of nerfing vox, it's a matter of nerfing the trader job. One job with significant drawbacks having a single piece of equipment which is equal to equipment other jobs get is far from ridiculous. The equipment fits the job and the vox only part mostly just prevents it being niggered away every round.

**Comment by Durgen** [comment]
Everyone is also conveniently forgetting that vox exosuits have slow down.

**Comment by yclatious** [comment]
>it's a matter of nerfing the trader job

God forbid the fact that the supposed underpowered underdogs dont have CE tier magboots so they can still fuck off to go vault raiding at a slightly slower pace, despite them having free roundstart jetpacks and canisters for mid spess flight refills to make the trip a lot quicker.

Heavens forbid the holy grail of jobs, Vox Trader, gets its metaclub with its panties in a twist over a nerf long due, long justified and long awaiting a coding, this is pure blasphemy, we must clearly shut this PR down immediately, for the sanctity of our feathery gypsie jew metaclub.

**Comment by kilozombie** [comment]
Give Vox Traders boots with light slowdown. Make a vault or two with pre-nerf Trader boots so that 1. traders can go space explore for them and 2. space explorers can find boots and sell them back to traders.

It's a lot to ask for somebody to make those vaults, so this PR doesn't have to do that, but it feels slightly crummy taking them away entirely. On the other hand, playing as a trader, this felt pretty crazy. If it were only light slowdown, it would actually make me think about when to turn on my boots, which is a good thing to encourage, in my mind.

And, of course, Raiders should have the pre-nerf ones as well.

**Comment by Exxion** [comment]
Even madder man

**Comment by Durgen** [comment]
fug

**Comment by Dusk-a** [comment]
I feel like its even slower than regular magboots due to the fact that you have to stay in your pressure suits that already slow you down or sacrifice your backpack. But thank god it has extra stomp damage that will never be used.

**Comment by Exxion** [comment]
I'm pretty sure engineering and mining hardsuits have the same slowdown as vox suits

**Comment by Sooose** [comment]
They don't have anywhere near vox pressure suit slowdown.

**Comment by Intigracy** [comment]
Yeah, the vox pressure suit slowdown is insane.

**Comment by Exxion** [comment]
You're right that vox suits have more, but it's 1.8 vs. 1.4

**Pull request merged**